### PR TITLE
Fixing some errors in plugins

### PR DESCRIPTION
**Pull request by rowdy-hero**

### CODE REVIEW

**Comment by Phyco-Ninja** [comment]
Fix a missing ```f``` [here ](https://github.com/rowdy-hero/Userge-Plugins/blob/patch-2/plugins/profile.py#L228) 🙂 u forgot to do that. 😂

**Comment by rowdy-hero** [comment]
> Fix a missing `f` [here ](https://github.com/rowdy-hero/Userge-Plugins/blob/patch-2/plugins/profile.py#L228)  u forgot to do that.

If I add f string, this may cause error maybe 😅

**Comment by Phyco-Ninja** [comment]
> > Fix a missing `f` [here ](https://github.com/rowdy-hero/Userge-Plugins/blob/patch-2/plugins/profile.py#L228)  u forgot to do that.
> 
> If I add f string, this may cause error maybe 😅

Lol what? How a``` f``` will  make error? 
Pay Respect to ```F```

**Comment by rowdy-hero** [comment]
> > > Fix a missing `f` [here ](https://github.com/rowdy-hero/Userge-Plugins/blob/patch-2/plugins/profile.py#L228)  u forgot to do that.
> > 
> > 
> > If I add f string, this may cause error maybe sweat_smile
> 
> Lol what? How a` f` will make error?
> Pay Respect to `F`

Dunno, but when I `f` , it course `f` string error

File: plugins/animesticker.py [hunk 1]
```diff
@@ -8,21 +8,40 @@
...
 @userge.on_cmd("sticker", about={
-    'header': "Its a joke with anime sticker",
-    'usage': "{tr}sticker [text | reply to message]",
-    'example': "{tr}sticker I am fool"}, allow_via_bot=False)
-async def anisti(message: Message):
+    'header': "Creates random anime sticker",
+    'flags': {
+        '-f': "To get only girls in anime",
+        '-ggl': "To get google search sticker",
+        'mock': "get mock text in sticker"},
```
Comment (rking32) [inline plugins/animesticker.py:15]: missing `-` in `mock`

File: plugins/figlet.py [hunk 2]
```diff
@@ -9,9 +9,8 @@
     'header': "Figlet",
     'description': "Make Fancy Style text using Figlet",
     'usage': "{tr}figlet font_name | [text | reply]",
-    'Available Fonts': "To know the font name list run command \n\n"
-    "<code>{tr}term pyfiglet --list_fonts</code>\n\n"
-    "<b>By the Way Plugin by:</b>\n\nAlone , @krishna_singhal"})
+    'Fonts': "[Check this link](https://telegra.ph/Figlet-List-Of-Fonts-07-03)"
+    " `to know available fonts`"})
```
Comment (rking32) [inline plugins/figlet.py:13]: markdown not supported in help section. can you convert this to html :+1:

**Pull request merged**